UI/GalaxySetupWnd.cpp

```diff
@@ -248,8 +244,11 @@ void GameRulesPanel::DoLayout() {
     m_tabs->SizeMove(MARGINS, ClientSize() - MARGINS);
 }
 
-void GameRulesPanel::Render()
-{ /*GG::FlatRectangle(UpperLeft(), LowerRight(), GG::CLR_CYAN, GG::CLR_GREEN, 1);*/ }
+void GameRulesPanel::Render() {
+    GG::FlatRectangle(m_tabs->CurrentWnd()->UpperLeft() - GG::Pt(GG::X(2), GG::Y0),
```

Comment:
?

Reply:
This is the same way that [GalaxySetupWnd makes a border for the galaxy image](https://github.com/wheals/freeorion/blob/86452b3fba50d7200b947a414d210568f861eb90/UI/GalaxySetupWnd.cpp#L1083), by drawing a black rectangle with the desired border. I'm not really sure if there's a more "correct" way to make a border in GG so I just copied the other code in the file.